contributing/write-general-writing-hints.md

```diff
@@ -14,63 +14,58 @@ Don't use complicated sentences. Not everyone reading the documentation will be
 
 ### Simple from technical standpoint
 
-Keep the doc simple from a '''technical standpoint''' too. Not everyone reading the documentation will be an experienced admin/user.
+Keep the doc simple from a **technical standpoint** too. Not everyone reading the documentation will be an experienced admin/user.
 
 ## Avoid shorthands
 
 They're just too informal.
 
 ### Bad example:
 
-```
-The update won't be available...
-```
+> The update won't be available...
 
 ### Good example:
 
-```
-The update will not be available...
-```
+> The update will not be available...
 
 ## Be direct
 
 Avoid passive voice.
 
 ### Bad example:
 
-```
-This can be fixed by...
-```
+> This can be fixed by...
 
 ### Good example:
 
-```
-You can fix this issue by...
-```
+> You can fix this issue by...
 
 ## Tell the user what is going on
 
 When you tell users to run a set of commands, tell them what is going on. This mostly applies to cases where the documentation text contains some procedure (a set of commands) to do some complicated stuff. When that happens, don't just put in the commands, but explain them.
 
 ### Bad example:
 
-```
-$ sudo systemctl disable firewalld
-$ sudo systemctl stop firewalld
-$ sudo cp PREUPGRADE_DIR/cleanconf/etc/sysconfig/iptables /etc/sysconfig/iptables
-etc.
-```
+> ```
+> $ sudo systemctl disable firewalld
+> $ sudo systemctl stop firewalld
+> $ sudo cp PREUPGRADE_DIR/cleanconf/etc/sysconfig/iptables /etc/sysconfig/iptables
```

Comment:
I'd rather not mix quot/paragraph and `code`. But as this is is actually showing block of examples it makes probable more sense to do so...